### ISSUE TITLE
Display training summary at the end of final epoch 

### ISSUE THREAD

**Issue by ls-da3m0ns** [PyTorchLightning/pytorch-lightning#4438]
## 🚀 Feature

It would be nice if trainer at the end of training prints complete summary of the training (train and val loss after each epoch)

**Comment by rohitgr7** [comment]
A custom callback can do this job.

**Comment by ananyahjha93** [comment]
@rohitgr7 do we have a custom callback for it or do you think it should be added in?

**Comment by rohitgr7** [comment]
@ananyahjha93 We don't have this right now. But if it's good to have such a callback, we can add it.

**Comment by JackCaster** [comment]
I'd like to have an example of such callback

**Comment by rohitgr7** [comment]
found something, this is already present in bolts: [PrintTableMetricsCallback](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/pl_bolts/callbacks/printing.py)
